modules/apps/27-interchain-accounts/controller/keeper/handshake.go

```diff
@@ -28,26 +28,35 @@ func (k Keeper) OnChanOpenInit(
 	chanCap *capabilitytypes.Capability,
 	counterparty channeltypes.Counterparty,
 	version string,
-) error {
+) (string, error) {
```

Comment:
what is the implication of changing an `OnChanOpenInit` signature like this? Are we free to do this without causing any issues?

Reply:
to add to that: in general, we would also update the ibc spec for changes to these fn sigs. But like @damiannolan said the calling func already has this signature.